src/cascadia/TerminalApp/TerminalPage.cpp

```diff
@@ -4570,6 +4588,21 @@ namespace winrt::TerminalApp::implementation
             // will eat focus.
             _paneResources.unfocusedBorderBrush = SolidColorBrush{ Colors::Black() };
         }
+
+        const auto broadcastColorKey = winrt::box_value(L"BroadcastPaneBorderColor");
+        if (res.HasKey(broadcastColorKey))
+        {
+            // MAKE SURE TO USE ThemeLookup
```

Comment:
Make sure to... why? 😅

Reply:
Oh that comment is explained at length like, two code blocks up

>             // MAKE SURE TO USE ThemeLookup, so that we get the correct resource for
>             // the requestedTheme, not just the value from the resources (which
>             // might not respect the settings' requested theme)